### PR TITLE
Only add user forks if GITHUB_USER (cmake or env) variable is set

### PR DESCRIPTION
**Pull request by eile**

### CODE REVIEW

**Comment by eile** [comment]
@tribal-tec @biddisco FYI

File: GitExternal.cmake [hunk 1]
```diff
@@ -100,13 +93,12 @@ function(GIT_EXTERNAL DIR REPO TAG)
   endif()
 
   # set up "user" remote for github forks
-  if(GIT_EXTERNAL_USER_FORK AND REPO MATCHES ".*github.com.*")
-    string(REGEX REPLACE "(.*github.com[\\/:]).*(\\/.*)"
-      "\\1${GIT_EXTERNAL_USER_FORK}\\2" GIT_EXTERNAL_USER_REPO ${REPO})
+  if(${GITHUB_USER AND REPO MATCHES ".*github.com.*")
```
Comment (rdumusc) [inline GitExternal.cmake]: missing closing }, but it think it should it be without the ${}

**Comment by rdumusc** [comment]
Very nice, +1 once the above is fixed!

File: GitExternal.cmake (hunk 1 above)
Comment (eile) [inline GitExternal.cmake]: Nice catch.

**Pull request merged**